pkg/operator/flags.go

```diff
@@ -10,6 +10,7 @@ const (
 	AzureSubnetsServiceEndpointManaged = "aro.azuresubnets.serviceendpoint.managed"
 	BannerEnabled                      = "aro.banner.enabled"
 	CheckerEnabled                     = "aro.checker.enabled"
+	CPMSEnabled                        = "aro.cpms.enabled"
```

Comment:
nit: this may become a confusing name in a world where we _do_ have OCP CPMS enabled en-masse for the fleet. We should probably use a name like `CPMSDisablerEnabled`

Reply:
I had that before and we decided to move away from that name due to it being confusing the other way round: https://github.com/Azure/ARO-RP/pull/3660/commits/c44d92065a0917ba667ec7f830453e746f226738